deepmd/utils/argcheck.py

```diff
@@ -522,6 +552,7 @@ def loss_variant_type_args():
     return Variant("type", 
                    [Argument("ener", dict, loss_ener()),
                     Argument("tensor", dict, loss_tensor()),
+                    Argument("masked_energy_forces", dict, loss_ener())
```

Comment:
Document should be added into this `Argument`, otherwise it's hard to understand what `masked_energy_forces` is.